### PR TITLE
Clean gatekeeper-uninstall-policy after the test

### PR DESCRIPTION
**Pull request by yiraeChristineKim**
non-clean **delete-gatekeeper-policy** affects other policies and re-tests, clusters.

### CODE REVIEW

File: test/integration/compliance_history_test.go [hunk 1]
```diff
@@ -440,10 +448,17 @@ var _ = Describe("GRC: [P1][Sev1][policy-grc] Test the compliance history API",
...
-				Expect(err).ToNot(HaveOccurred())
-			}
+			_, err = common.OcHub(
+				"delete",
+				"ns",
+				gkTargetNS,
+			)
+
+			Eventually(func() bool {
+				_, err := clientHub.CoreV1().Namespaces().Get(ctx, gkTargetNS, metav1.GetOptions{})
```
Comment (yiraeChristineKim) [inline test/integration/compliance_history_test.go:458]: To reduce flaky test result

**Comment by yiraeChristineKim** [comment]
I tested 4 times. All was ok

**Review by JustinKuli — APPROVED** [review]

**Pull request merged**